api/routes/onboarding.py

```diff
@@ -1,4 +1,10 @@
-from flask import Blueprint
+from flask import Blueprint, redirect, render_template, request, url_for
+import database.queries as queries 
```

Comment:
Rather specify the method you want to import from the file rather than the whole file. For example from database.queries import check_if_number_exists